app/models/location.rb

```diff
@@ -15,10 +15,10 @@ def inn
   end
 
   def blacksmith
-    @blacksmith ||= stores.stores.find{|store| store.category.name == "blacksmith"}
+    @blacksmith ||= stores.find{|store| store.category.name == "blacksmith"}
   end
 
   def apothecary
-    @apothecary ||= stores.stores.find{|store| store.category.name == "apothecary"}
+    @apothecary ||= stores.find{|store| store.category.name == "apothecary"}
```

Comment:
Prefer `detect` over `find`.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.